eZ/Publish/Core/Persistence/Legacy/Content/Location/Trash/Handler.php

```diff
@@ -8,13 +8,15 @@
  */
 namespace eZ\Publish\Core\Persistence\Legacy\Content\Location\Trash;
 
-use eZ\Publish\SPI\Persistence\Content\Location\Trashed;
-use eZ\Publish\SPI\Persistence\Content\Location\Trash\Handler as BaseTrashHandler;
-use eZ\Publish\Core\Persistence\Legacy\Content\Handler as ContentHandler;
 use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
-use eZ\Publish\Core\Persistence\Legacy\Content\Location\Handler as LocationHandler;
+use eZ\Publish\Core\Persistence\Legacy\Content\Handler as ContentHandler;
 use eZ\Publish\Core\Persistence\Legacy\Content\Location\Gateway as LocationGateway;
+use eZ\Publish\Core\Persistence\Legacy\Content\Location\Handler as LocationHandler;
 use eZ\Publish\Core\Persistence\Legacy\Content\Location\Mapper as LocationMapper;
+use eZ\Publish\SPI\Persistence\Content\Location\Trash\Handler as BaseTrashHandler;
+use eZ\Publish\SPI\Persistence\Content\Location\Trash\TrashedItemDeleteResult;
+use eZ\Publish\SPI\Persistence\Content\Location\Trash\TrashedItemDeleteResultList;
```

Comment:
FIY: PHPStorm did some order change here. I will fix it before finalizing the PR.